conans/client/graph/range_resolver.py

```diff
@@ -25,12 +25,16 @@ def resolve(self, require, base_conanref, remotes, update):
             require.ref = previous_ref
             return
 
+        # If the conf is None, do whatever the range specifies
+        # True: Force prerelease resolving, False: Force no prerelease resolving
+        resolve_prereleases = self._cache.new_config.get('core.ranges_resolve_prereleases', default=None, check_type=bool)
```

Comment:
confs should be ``xxx.yyy:conf=value``

Reply:
`core.version_ranges:resolve_prereleases` it is then